src/main/java/io/kestra/plugin/redis/ListPush.java

```diff
@@ -68,16 +68,30 @@ public Output run(RunContext runContext) throws Exception {
         try (RedisFactory factory = this.redisFactory(runContext)) {
 
             Integer count;
-            if (this.from instanceof String fromUrl) {
-                URI from = new URI(runContext.render(fromUrl));
-                try (BufferedReader inputStream = new BufferedReader(new InputStreamReader(runContext.uriToInputStream(from)))) {
+
+            Object from = null;
+            if (this.from instanceof String fromString) {
+                String renderedFrom = runContext.render(fromString);
+                try {
+                    ObjectMapper mapper = new ObjectMapper();
```

Comment:
You should retrieve an instance of an Object Mapper as a constant using our Jackson helper to have a pre-configured mapper.